web/client.js

```diff
@@ -1,5 +1,13 @@
+import * as d3 from 'd3';
+import BootstrapDialog from 'bootstrap3-dialog';
+import BootstrapMenu from 'bootstrap-menu';
+import Sortable from 'sortablejs';
+import * as $ from 'jquery';
+require('bootstrap-select');
+require('d3-graphviz');
+
 $(function () {
-    settings = {};
+    var settings = {};
```

Comment:
This was an intentional use of the global object to allow for tinkering of the settings in F12.  Can you revert this first?

Reply:
This commit isn't supposed to be in this PR, however cluttering up the global scope with a variable called settings doesn't seem wise.

If you want it to be a global variable, I can explicitly add a global variable, but webpack balks if you implicitly do so.